Python/Product/PythonTools/PythonTools/Repl/Completion/AsyncCompletionSource.cs

```diff
@@ -249,13 +250,16 @@ internal class AsyncCompletionSource : IAsyncCompletionUniversalSource {
                             buffer.Delete(deleteSpan);
                         }
 
-                        if (protocolItem.TextEdit != null) {
-                            Utilities.ApplyTextEdit(protocolItem.TextEdit, triggerLocation.Snapshot, buffer);
-                        } else if (protocolItem.InsertText != null) {
-                            buffer.Replace(session.ApplicableToSpan.GetSpan(buffer.CurrentSnapshot), protocolItem.InsertText);
-                        } else if (protocolItem.Label != null) {
-                            buffer.Replace(session.ApplicableToSpan.GetSpan(buffer.CurrentSnapshot), protocolItem.Label);
-                        }
+                        if (protocolItem.TextEdit != null && protocolItem.TextEdit.HasValue) {
```

Comment:
what need to check both protocolItem.TextEdit != null and protocolItem.TextEdit.HasValue here? Doesn't not null mean it has value?